### ISSUE TITLE
[benchmarks] 2009 Mac Pro with NVIDIA 980 Ti

### ISSUE THREAD

**Issue by webbp** [DTolm/VkFFT#37]
2009 Mac Pro 4,1->5,1 12-core 3.06GHz Xeon 96GB RAM macOS 10.13.6 with NVIDIA 980 Ti
VkFFT commit 6947846
Setup:
```
sudo port install MoltenVK vulkan-headers vulkan-loader vulkan-tools vulkan-validationlayers
git clone https://github.com/DTolm/VkFFT; cd VkFFT; mkdir build; cd build; cmake ..; make -24
N=0; time ./Vulkan_FFT -vkfft $N >$N.log 2>$N.err
N=1 ...
```
Results
```
0 0h:01m:51s Benchmark score VkFFT: 9928303
1 crashes
2 crashes
3 0h:00m:49s Benchmark score VkFFT: 42851
4 0h:01m:14s Benchmark score VkFFT: 54713
5 0h:01m:48s Benchmark score VkFFT: 9320665
6 0h:00m:44s Benchmark score VkFFT: 15106
```
[0.log](https://github.com/DTolm/VkFFT/files/6497946/0.log)
[0.err.log](https://github.com/DTolm/VkFFT/files/6497945/0.err.log)
[1.log](https://github.com/DTolm/VkFFT/files/6497969/1.log)
[1.err.log](https://github.com/DTolm/VkFFT/files/6497947/1.err.log)
2.log [empty]
[2.err.log](https://github.com/DTolm/VkFFT/files/6497949/2.err.log)
[3.log](https://github.com/DTolm/VkFFT/files/6497953/3.log)
[3.err.log](https://github.com/DTolm/VkFFT/files/6497951/3.err.log)
[4.log](https://github.com/DTolm/VkFFT/files/6497955/4.log)
[4.err.log](https://github.com/DTolm/VkFFT/files/6497954/4.err.log)
[5.log](https://github.com/DTolm/VkFFT/files/6497957/5.log)
[5.err.log](https://github.com/DTolm/VkFFT/files/6497956/5.err.log)
[6.log](https://github.com/DTolm/VkFFT/files/6497959/6.log)
[6.err.log](https://github.com/DTolm/VkFFT/files/6497958/6.err.log)

**Comment by DTolm** [comment]
Hello,
Judging by the reports, there is some issue with push constants binding in the MoltenVk version you are using. Can you update it to more recent one and try again? (1.0.27 was released two years ago)
Best regards,
Dmitrii

**Comment by DTolm** [comment]
Closed, as the issue has not been active.

**Comment by webbp** [comment]
I agree closing the issue is the best move here, and apologies for not responding to your request. I did attempt to build a newer MoltenVk, but was not successful, as recent versions require xcode 11, which is not available in macOS 10.13.6.
